content/influxdb/cloud/reference/glossary.md

```diff
@@ -9,4 +9,1216 @@ menu:
 influxdb/cloud/tags: [glossary]
 ---
 
-{{< duplicate-oss >}}
```

Comment:
I don't think we should duplicate this content. I know there are cloud specific details here, but I think using `oss-only` and `cloud-only` shortcodes will make our glossary easier to maintain.

Reply:
thanks, thought I removed this

Reply:
Sorry, my comment was confusing. I think we should keep the `duplicate-oss` shortcode rather than actually duplicating the content in cloud.